airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_rule_libraries.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties" : {
```

Comment:
Please make sure your schema follow https://docs.airbyte.com/connector-development/schema-reference

Reply:
Done, sorry about that.